### PR TITLE
feat(Minecraft Menu): add presence

### PR DESCRIPTION
**Pull request by DarkVillager**
**Describe the changes in this pull request:**
- Added Full Site Support
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

</details>

### CODE REVIEW

**Comment by Slowlife01** [comment]
The domain must be at least 2 months old.

**Comment by DarkVillager** [comment]
> The domain must be at least 2 months old.

Lemme see what the owner of the site has to say, I think ur right tho.

**Comment by DarkVillager** [comment]
@EncryptedDev @Bas950 could u reopen this please?

File: websites/M/Minecraft Menu/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Dark_Ville",
+		"id": "638080361179512853"
+	},
+	"service": "Minecraft Menu",
+	"description": {
+		"nl": "Minecraft Menu is een plek om de Beste Minecraft Server te vinden in 2022 voor Minecraft.",
```
Comment (EncryptedDev) [inline websites/M/Minecraft Menu/dist/metadata.json:9]: @Bas950

File: websites/M/Minecraft Menu/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+		"en": "Minecraft Menu is a place to find the Best Minecraft Servers in 2022 on Minecraft."
+	},
+	"url": [
+		"minecraft.menu",
+		"www.minecraft.menu"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/BlAb5Pg.png",
```
Comment (EncryptedDev) [inline websites/M/Minecraft Menu/dist/metadata.json:17]: This scales poorly on the website, resize the image to be as close to 512x512 without messing up the quality

File: websites/M/Minecraft Menu/presence.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+		title = document.querySelector(
+			"body > div.pusher > section > div.ui.stackable.grid > div.six.wide.column > table > thead > tr > th"
+		);
+		presenceData.details = title.textContent.trim();
+		presenceData.state = document.querySelector(
+			"[class='active item']"
+		).textContent;
+		presenceData.buttons = [
+			{
+				label: "View server",
```
Comment (EncryptedDev) [inline websites/M/Minecraft Menu/presence.ts:36]: Use [Title Case](https://titlecaseconverter.com/rules/#NYT)
```suggestion
				label: "View Server",
```

File: websites/M/Minecraft Menu/presence.ts [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+				url: document.location.href,
```
Comment (EncryptedDev) [inline websites/M/Minecraft Menu/presence.ts:37]:
```suggestion
				url: href,
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Hiya - please update the button changes for all buttons

File: websites/M/Minecraft Menu/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,45 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Dark_Ville",
+		"id": "638080361179512853"
+	},
+	"service": "Minecraft Menu",
+	"description": {
+		"nl": "Minecraft Menu is in 2022 een plek om de Beste Minecraft Servers te vinden.",
```
Comment (EncryptedDev) [inline websites/M/Minecraft Menu/dist/metadata.json:9]: @Bas950

File: websites/M/Minecraft Menu/presence.ts [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+		},
+		search = document.querySelector<HTMLInputElement>(
+			"body > div.pusher > div.site-nav.clearfix > div > header > nav > div.right.menu > div > div.ui.right.action.left.icon.input > input"
+		),
+		{ pathname, href } = document.location,
+		[privacy, buttons] = await Promise.all([
+			presence.getSetting<boolean>("privacy"),
+			presence.getSetting<boolean>("buttons"),
+		]);
+	if (privacy) presenceData.details = "Browsing...";
```
Comment (EncryptedDev) [inline websites/M/Minecraft Menu/presence.ts:19]:
```suggestion
	if (privacy) presenceData.details = "Browsing";
```

File: websites/M/Minecraft Menu/presence.ts [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+	else if (search.value) {
+		presenceData.details = "Searching for:";
```
Comment (EncryptedDev) [inline websites/M/Minecraft Menu/presence.ts:21]:
```suggestion
		presenceData.details = "Searching for";
```

**Review by EncryptedDev — COMMENTED** [review]
Two final things from me

File: websites/M/Minecraft Menu/dist/metadata.json (hunk 5 above)
Comment (Bas950) [inline websites/M/Minecraft Menu/dist/metadata.json:9]:
```suggestion
		"nl": "Minecraft Menu is de beste plek in 2022 om Minecraft-servers te vinden.",
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**